### PR TITLE
Configuration.md: Remove a trailing comma

### PR DESCRIPTION
**Pull request by fviernau**
Signed-off-by: Frank Viernau <frank.viernau@here.com>

### CODE REVIEW

File: docs/Configuration.md [hunk 1]
```diff
@@ -179,7 +179,7 @@ e.g.:
     start_lines: "3"
     line_count: 11
     detected_license: "GPL-2.0-only"
-    reason: CODE,
+    reason: CODE
```
Comment (sschuberth) [inline docs/Configuration.md]: As mentioned [here](https://github.com/heremaps/oss-review-toolkit/pull/1856#discussion_r330069585), I believe we should quote this (even if not technically required in this case).

Comment (fviernau) [reply docs/Configuration.md]: Done.

**Comment by sschuberth** [comment]
> Use quotes even if not technically required and remove the trailing
comma.

Nit, *if* you already mention that quotes are not required, it would make sense to explain why we still add them. How about: "Remove the erroneous trailing comma and quote the enum name for consistency with how the value would be serialized."

**Comment by fviernau** [comment]
> Nit, _if_ you already mention that quotes are not required, it would make sense to explain why we still add them. How about: "Remove the erroneous trailing comma and quote the enum name for consistency with how the value would be serialized."

Done.

**Pull request merged**